lib/bootstrap_form/form_builder.rb

```diff
@@ -198,10 +198,10 @@ def form_group(*args, &block)
         control.concat(generate_icon(options[:icon])) if options[:icon]
 
         if get_group_layout(options[:layout]) == :horizontal
-          control_class = (options[:control_col] || control_col.clone)
+          control_class = options[:control_col] || control_col
```

Comment:
This offers a slight memory improvement since it's only creating new string objects as needed, instead of cloning every time.